builders/java/README.md

```diff
@@ -1,5 +1,12 @@
 ## Build builder of java
 
+
+Required soft on your linux
```

Comment:
```suggestion
You'll need to install the following prerequisites first"
```

Reply:
@gzchen008 You'll need to make the change below as I suggested 2 weeks ago:
![image](https://github.com/OpenFunction/builder/assets/18525465/70cd19d2-1bd1-435f-a3e0-ee16a9c9b5f3)